src/Monolog/Formatter/LogglyFormatter.php

```diff
@@ -37,7 +37,7 @@ public function __construct($batchMode = self::BATCH_MODE_NEWLINES, $appendNewli
      */
     public function format(array $record)
     {
-        if (isset($record["datetime"]) && ($record["datetime"] instanceof \DateTime)) {
+        if (isset($record["datetime"]) && ($record["datetime"] instanceof \DateTimeInterface)) {
```

Comment:
This won't work with PHP 5.3 and 5.4. The `DateTimeInterface` was introduced in PHP 5.5.